aws/resource_aws_db_option_group.go

```diff
@@ -273,12 +273,24 @@ func resourceAwsDbOptionGroupUpdate(d *schema.ResourceData, meta interface{}) er
 		}
 
 		log.Printf("[DEBUG] Modify DB Option Group: %s", modifyOpts)
-		_, err = rdsconn.ModifyOptionGroup(modifyOpts)
+
+		err = resource.Retry(5*time.Minute, func() *resource.RetryError {
```

Comment:
IAM eventual consistency is usually on the magnitude of 1-2 minutes (very rarely the latter). Can you please lower this retry timeout to prevent bad user experience when there might be an actual issue?